components/annot-tab.vue

```diff
@@ -1,36 +1,23 @@
 <template>
 
-    <div>
+    <div :data-cy="activeCategory">
 
         <!-- Explanation text for this annotation tab -->
-        <annot-explanation :explanation="details.explanation" :index="details.id" :data-cy="'annot-expl-' + title" />
+        <annot-explanation :data-cy="('annot-expl-' + activeCategory)" />
```

Comment:
@surchs Given how the other two child components receive `activeCategory` as props, the `annot-explanation` should receive it as well for when we each annotation component has a different set of instructions.

Reply:
Aha, yes, great catch. That's a mistake and I'll fix it!